env.template

```diff
@@ -1,7 +1,8 @@
-PLUGINS="example"
+PLUGINS="example google-analytics"
 WP_TOKEN=
 NODE_ENV=development
 THEME=example
 TRANSLATIONS=
 API_URL=http://127.0.0.1:5000/api/3/action/
 WP_URL=http://127.0.0.1:6000
+GA_ID=UA-143981871-1
```

Comment:
Not sure if the GA_ID belongs in the public commit? Do we want this to be enabled by default? Could be:
```
PLUGINS=example
WP_TOKEN=
NODE_ENV=development
THEME=example
TRANSLATIONS=
API_URL=http://127.0.0.1:5000/api/3/action/
WP_URL=http://127.0.0.1:6000
GA_ID=
```

Reply:
Requested changes have been implemented.